### PR TITLE
Added routes for resume backend functions

### PR DESCRIPTION
**Pull request by WilliamSterner**
Added API routes for Owen's resume backend

### CODE REVIEW

**Review by EmilianoC02 — APPROVED** [review]
Did we check if we have to call these by stating, for example, resume.resume.addExperience(), or was that figure out?

File: backend/main.py [hunk 1]
```diff
@@ -174,6 +175,109 @@ def createAccount():
...
 
+
+@stuAPI.route('/api/addResumeExperience', methods=['POST'])
+def addResumeExperience():
+    resume = Resume():
+    data = request.get_json()
+    user_id = data['user_id']
+    company = data['company']
+    role = data['role']
+    start_date = data["start_date"]
```
Comment (Grayson34) [inline backend/main.py:186]: why does this one have double quotes

File: backend/main.py [hunk 2]
```diff
@@ -174,6 +175,109 @@ def createAccount():
     email = data['email']
     return handleCreateAccount.sign_up(username, email, password, confirm_password)
 
+
+@stuAPI.route('/api/addResumeExperience', methods=['POST'])
+def addResumeExperience():
+    resume = Resume():
```
Comment (Grayson34) [inline backend/main.py:181]: why is this being created

**Review by Grayson34 — APPROVED** [review]
Looks good.
Need to add later routes 
- generate_resume
- valid_resume_data
- delete_resume_data

**Pull request merged**